src/test/java/uk/gov/companieshouse/mapper/email/EmailMapperTest.java

```diff
@@ -8,9 +8,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static uk.gov.companieshouse.fixtures.EmailFixtures.EMAIL_ADDRESS;
-import static uk.gov.companieshouse.fixtures.EmailFixtures.EMAIL_DATA;
-import static uk.gov.companieshouse.fixtures.EmailFixtures.MESSAGE_TYPE;
+import static uk.gov.companieshouse.fixtures.EmailFixtures.*;
```

Comment:
👀

Reply:
These are automatic IDE things

Reply:
I know it's a bit annoying. Can disable that following https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class

Reply:
done